### PR TITLE
stats, executor: fix drop stats not working

### PR DESCRIPTION
**Pull request by lamxTyler**

### CODE REVIEW

File: statistics/ddl.go [hunk 1]
```diff
@@ -34,6 +34,9 @@ func (h *Handle) HandleDDLEvent(t *util.Event) error {
 		return h.insertTableStats2KV(t.TableInfo)
 	case model.ActionAddColumn:
 		return h.insertColStats2KV(t.TableInfo.ID, t.ColumnInfo)
+	// For drop stats statement.
+	case model.ActionDropTable:
```
Comment (shenli) [inline statistics/ddl.go]: We should consider `Truncate Table` and `Drop database`.

Comment (lamxTyler) [reply statistics/ddl.go]: They will be handled by garbage collection.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**